code/controllers/subsystems/initialization/map_finalization.dm

```diff
@@ -149,9 +173,9 @@
 	qdel(maploader)
 
 /datum/controller/subsystem/finalize/proc/generate_contact_report()
-	if(!selected_ruin)
+	if(!selected_mission)
 		return
-	var/report_text = selected_ruin.get_contact_report()
+	var/report_text = selected_mission.get_contact_report()
 	for(var/obj/effect/landmark/C in landmarks_list)
 		if(C.name == "Space Ruin Paper")
```

Comment:
```suggestion
		if(C.name == "Space Ruin Paper")
```

Should this be 'Mission Paper' now or something since you're renaming everything Ruin > Mission? Not that important.